api/mailer/Util.js

```diff
@@ -0,0 +1,69 @@
+const axios = require('axios');
+const bcrypt = require('bcryptjs');
+const User = require('../models/User.js');
+
+async function validateVerificationEmail(){
+  let status = '';
+  await axios
+    .post('api/mailer/sendVerificationEmail')
+    .then(res =>{
+      status = res.data;
+    })
+    .catch(err => {
+      status = err.data;
+    });
+  return status;
+}
+
+async function generateHashedId(email) {
+  return new Promise( async (resolve, reject) => {
+    // Find a user in the DB
+    User.findOne({ email }, function(error, result) {
+      // Return an error if there's a problem (e.g. email is null)
+      if (error) {
+        reject(new Error('Bad request'));
+      }
+
+      // If no user found, return an error
+      if (!result) {
+        reject(new Error('User not found'));
+      }
+
+      // Use the unique _id from the User schema
+      let hashedId = String(result._id);
+      // Generate a salt and created a hashed value of the _id using
+      // bcrypts library
+      bcrypt.genSalt(10, function(error, salt) {
+        if (error) {
+          // reject('Bcrypt failed')
+          reject(new Error('Bcrypt failed'));
+        }
+
+        bcrypt.hash(hashedId, salt, function(error, hash) {
+          if (error) {
+            reject(new Error('Bcrypt failed'));
+          }
+          hashedId = hash;
+          resolve(hashedId);
+        });
+      });
+    });
+  });
+}
+// async function generateHashedId(){
+//   let hashedID = ""
+//   await axios
+//     .post('api/mailer/generateHashedId')
+//     .then(res =>{
+//       hashedID = res;
+//     })
+//     .catch(err => {
+//      hashedID = err;
+//     });
+//   return hashedID;
+// }
```

Comment:
Are we going to re-include this commented out function later. If not, good idea to remove it xd